Surface_mesh_simplification/doc/Surface_mesh_simplification/Surface_mesh_simplification.txt

```diff
@@ -403,8 +417,8 @@ betwen 2006 and 2009.
 Andreas Fabri added the `Bounded_normal_change_placement` functionality in \cgal 4.11.
```

Comment:
As you are at it, can you put the SMS namespace in front so that there will be a link. And maybe change to `Bounded_normal_change_filter` as the placement is deprecated already.